src/shipping/consignment.ts

```diff
@@ -22,7 +23,7 @@ export type ConsignmentRequestBody =
 
 export interface ConsignmentCreateRequestBody {
     address?: AddressRequestBody;
-    shippingAddress?: AddressRequestBody;
+    shippingAddress?: AddressRequestBody | AddressData;
```

Comment:
I think that you can add `email?: string` to the `AddressRequestBody` and remove `AddressData` from this line. `AddressRequestBody` already have all the data in `AddressData`.

Reply:
AddressRequestBody has more required fields than i need to get response with data in this case

Reply:
Perhaps you can make them optional?

Reply:
Or it's better check by Checkout team

Reply:
It is used in many different places so i would need to cheat check all of them and fix possible issues